packages/microsoft_sqlserver/manifest.yml

```
@@ -1,33 +1,74 @@
 format_version: 1.0.0
 name: microsoft_sqlserver
 title: "Microsoft SQL Server"
-version: 1.0.0
+version: 1.0.1
```

As the kibana constraint is being changed, please change the version to `1.1.0`, so `1.0.X` can be used to release fixes for older versions of the stack.
```suggestion
version: 1.1.0
```

---

Done